tests/test_image_types.py

```diff
@@ -165,6 +169,23 @@ def test_src_set_invalid_format(self):
         self.assertEqual(len(results["errors"]), 1)
         self.assertIn("Format must be either 'jpeg'", results["errors"][0]["message"])
 
+    @skipIf(
+        WAGTAIL_VERSION < (5, 2),
+        "Wagtail 5.2 introduced changes to this error message.",
+    )
+    def test_src_set_invalid_format_wagtail_above_5_2(self):
```

Comment:
Please add a docstring detailing this test, and it's relevance with the other test.